react/Button/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import cx from 'classnames'
 
-export default function (props) {
+export default (props) => {
```

Comment:
I move the code into a specific folder to include `README.md` file (that I did not write).
It may not change anything in your third-party project because you may `import {Button} from 'cozy-ui/react'`

Reply:
For info, another way to go is to write the documentation in `Button.md`. This way you do not have to create a folder.

Reply:
Oh, right, but if every components are in folders, maybe it is a good think to follow this habit.